### PR TITLE
Fix make lint-docstrings

### PR DESCRIPTION
**Pull request by m-vdb**
**Proposed changes**:
- ...

**Status (please check what you already did)**:
- [ ] added some tests for the functionality
- [ ] updated the documentation
- [ ] updated the changelog (please check [changelog](https://github.com/RasaHQ/rasa/tree/main/changelog) for instructions)
- [ ] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa#code-style) for instructions)

### CODE REVIEW

File: scripts/lint_python_docstrings.sh [hunk 1]
```diff
@@ -4,21 +4,28 @@ set -euo pipefail
 # Lint docstrings only against the the diff to avoid too many errors.
 # Check only production code. Ignore other other errors which are captured by `lint`
 
+echo "1-here"
 # Compare against `main` if no branch was provided
 BRANCH="${1:-main}"
+
+echo "2-here"
 # Diff of committed changes (shows only changes introduced by your branch
 FILES_WITH_DIFF=`git diff $BRANCH...HEAD --name-only -- rasa | grep -e '\.py$' | xargs echo -n`
```
Comment (m-vdb) [inline scripts/lint_python_docstrings.sh:13]: this seems to fail

Comment (m-vdb) [reply scripts/lint_python_docstrings.sh:13]: that's the offending commit https://github.com/RasaHQ/rasa/commit/4aefb5fce536a2e33a9573c887c00d84a8c93012

**Review by sanchariGr — APPROVED** [review]
🎉

**Review by Shmuma — COMMENTED** [review]
Looks good! Thanks for the fix

**Pull request merged**